common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -265,7 +265,10 @@ public class CommonValue {
                         + MLModel.CONNECTOR_FIELD
                         + "\": {" + ML_CONNECTOR_INDEX_FIELDS + "    }\n},"
                         + USER_FIELD_MAPPING
-                        + "    }\n"
+                        + "    },\n"
+                        + "      \""
+                        + MLModel.GUARDRAILS_FIELD
+                        + "\" : {\"type\": \"flat_object\"},\n"
```

Comment:
Any limitation for using flat_object ? like how many fields we can save ? Is it possible to support thousands of stop words/regex?